superset/views/csv_import.py

```diff
@@ -211,7 +218,23 @@ def _create_database(self, db_name: str, db_flavor="sqlite"):
                 + "@localhost/"
                 + db_name
             )
-            engine = sqlalchemy.create_engine(url)
+            # TODO if there is time look for a better method to do this, otherwise document it
```

Comment:
Don't check in TODO's in master, create an issue and point to this line of code in the issue